detox/src/devices/runtime/drivers/ios/allowNetworkPermissionsXCUITest.scpt

```diff
@@ -0,0 +1,14 @@
+tell application "System Events"
+    set frontmost of process "UserNotificationCenter" to true
+    tell process "UserNotificationCenter"
+        repeat until (exists button "Allow" of window 1)
```

Comment:
I just wonder if I have anything other hanging in UserNotificationCenter, wouldn't it block this loop (because it looks at `window 1`)?